includes/Core/Admin/Standalone.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Admin\Standalone
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Admin;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Assets\Stylesheet;
+
+/**
+ * Class managing standlone mode.
+ *
+ * @since 1.0.0
+ * @access private
+ * @ignore
+ */
+final class Standalone {
+
+	/**
+	 * Plugin context.
+	 *
+	 * @since NEXT
+	 *
+	 * @var Context
+	 */
+	private $context;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since NEXT
+	 *
+	 * @param Context $context Plugin context.
+	 */
+	public function __construct( Context $context ) {
+
+		$this->context = $context;
+
+		$this->standalone_mode();
+
+	}
+
+	/**
+	 * Standalone mode
+	 *
+	 * @since NEXT
+	 */
+	public function standalone_mode() {
```

Comment:
Let's call this method `register` instead - that's a convention established in all classes in the plugin which add WordPress hooks. The doc-block description should be `Registers functionality through WordPress hooks.`